hal/src/electron/platform_ncp_electron.cpp

```diff
@@ -0,0 +1,62 @@
+/*
```

Comment:
Should I create a similar file for Photon just to be consistent?

Reply:
Just for consistency, but you might want to just hardcode `USI09` for Photon and `USI14` for P1.

Reply:
From the documentation, the Photon and P1 use a BCM43362 module. How does this relate to `PLATFORM_NCP_BROADCOM_BCM9WCDUSI09` and `PLATFORM_NCP_BROADCOM_BCM9WCDUSI14`?

It makes sense to add `/hal/src/photon/platform_ncp_photon.cpp` for photon, but what about P1? Will it use the same code? Should I check for `PLATFORM` in `platform_primary_ncp_identifier()` and return `USI09` on photon platform and `USI14` on P1 platform? (Also, I dont have a photon or P1 board to test with)

Reply:
BCM43362 is the Cypress/Broadcom IC. What is used on Photon and P1s are modules with additional components and their own differences. The one on Photon is called `BCM9WCDUSI09` the one on P1 is `BCM9WCDUSI14`.

Yes, you should implement it once for both Photon and P1 and use a conditional on e.g. `PLATFORM_ID == PLATFORM_PHOTON`.